src/platform/qemu-riscv64-virt/platform.mk

```
@@ -5,6 +5,8 @@
 ARCH:=riscv
 # CPU definition
 CPU:=
+# Interrupt controller definition
+IRQC:=plic
```

Are you able to override this from the command line? I think we need something like:

```suggestion
IRQC?=plic
```

So we can easily do something like:

```
make PLATFORM=qemu-riscv64-virt IRQC=aplic CONFIG=...
```

---

I have tested it, and this is my conclusion: Both types of assignment allow the command line input to override the variable (which takes priority). 
From [6.2.4 Conditional Variable Assignment](https://www.gnu.org/software/make/manual/html_node/Flavors.html#Flavors) I see that with "?" the variable can only be defined if not defined yet. I believe that makes sense to use "?" either way, making that define to be the only one that can change the IRQC variable (aside from the command line). Likewise, I'm changing it.

---

Yes you are right. Might suggestion is only needed if we want to override it using an environment variable. I don't think this is needed right now.